src/app/clusters/smoke-co-alarm-server/smoke-co-alarm-server.h

```diff
@@ -102,6 +104,15 @@ class SmokeCoAlarmServer
 
     inline bool SupportsCOAlarm(chip::EndpointId endpointId) { return GetFeatures(endpointId).Has(Feature::kCoAlarm); }
 
+    /**
+     * @brief ExpressedState priority, the priority order of conditions is determined by the manufacturer
```

Comment:
If it's determined by the manufacturer, why is it hardcoded here?  This does not make sense.